lib/webResource.ts

```diff
@@ -298,65 +298,90 @@ export class WebResource {
- * @param {string} [options.baseUrl] The base url of the request. Default value is: "https://management.azure.com". This is applicable only with
- * options.pathTemplate. If you are providing options.url then it is expected that you provide the complete url.
- * @param {object} [options.pathParameters] A dictionary of path parameters that need to be replaced with actual values in the pathTemplate.
- * Here the key is the "path-parameter-name" and the value is the "path-parameter-value".
- * The "path-parameter-value" can be of type "string"  or it can be of type "object".
- * The "object" format should be used when you want to skip url encoding. While using the object format,
- * the object must have a property named value which provides the "path-parameter-value".
- * Example:
- *    - path-parameter-value in "object" format: { "path-parameter-name": { value: "path-parameter-value", skipUrlEncoding: true } }
- *    - path-parameter-value in "string" format: { "path-parameter-name": "path-parameter-value" }.
- * @param {object} [options.headers] A dictionary of request headers that need to be applied to the request.
- * Here the key is the "header-name" and the value is the "header-value". The header-value MUST be of type string.
- *  - ContentType must be provided with the key name as "Content-Type". Default value "application/json; charset=utf-8".
- *  - "Transfer-Encoding" is set to "chunked" by default if "options.bodyIsStream" is set to true.
- *  - "Content-Type" is set to "application/octet-stream" by default if "options.bodyIsStream" is set to true.
- *  - "accept-language" by default is set to "en-US"
- *  - "x-ms-client-request-id" by default is set to a new Guid. To not generate a guid for the request, please set options.disableClientRequestId to true
- * @param {boolean} [options.disableClientRequestId] When set to true, instructs the client to not set "x-ms-client-request-id" header to a new Guid().
- * @param {object|string|boolean|array|number|null|undefined} [options.body] - The request body. It can be of any type. This method will JSON.stringify() the request body.
- * @param {object} [options.serializationMapper] - Provides information on how to serialize the request body.
- * @param {object} [options.mappers] - A dictionary of mappers that may be used while [de]serialization.
- * @param {object} [options.deserializationMapper] - Provides information on how to deserialize the response body.
- * @param {boolean} [options.disableJsonStringifyOnBody] - Indicates whether this method should JSON.stringify() the request body. Default value: false.
- * @param {boolean} [options.bodyIsStream] - Indicates whether the request body is a stream (useful for file upload scenarios).
- * @returns {object} WebResource Returns the prepared WebResource (HTTP Request) object that needs to be given to the request pipeline.
- */
 export interface RequestPrepareOptions {
+  /**
```

Comment:
Very good

Reply:
Added in https://github.com/Azure/ms-rest-js/pull/166/commits/6898d391c4fc522faea07c70a358da2867a99791.